src/main/java/in/org/projecteka/hdaf/link/link/Link.java

```diff
@@ -17,7 +18,11 @@
 import static in.org.projecteka.hdaf.link.link.Transformer.toHIPPatient;
 
 public class Link {
-
+    public static final ClientError HIP_NOT_FOUND = new ClientError(
```

Comment:
This can be a static method in ClientError class itself or separate Excpetion Builder Class. Refer https://github.com/ProjectEKA/hip-service/blob/master/hip-service/Discovery/Patient/Ranker/RankBuilder.cs
https://github.com/ProjectEKA/hip-service/blob/master/hip-service/Discovery/Patient/Ranker/PatientWithRankBuilder.cs